src/core/include/cesium/omniverse/GltfAccessors.h

```diff
@@ -89,7 +91,7 @@ class VertexColorsAccessor {
     VertexColorsAccessor(const CesiumGltf::AccessorView<glm::fvec3>& float32Vec3View);
     VertexColorsAccessor(const CesiumGltf::AccessorView<glm::fvec4>& float32Vec4View);
 
-    void fill(const gsl::span<glm::fvec4>& values) const;
+    void fill(const gsl::span<glm::fvec4>& values, uint64_t repeat = 1) const;
```

Comment:
NIT: We should probably add documentation here while we're at it.

Reply:
Done, but only for this function. We'll need to do a doc pass at some other point.